admob/src_java/com/google/firebase/admob/internal/cpp/BannerViewHelper.java

```diff
@@ -175,9 +173,9 @@ public void run() {
             notifyStateChanged(
                 mBannerViewInternalPtr, ConstantsHelper.AD_VIEW_CHANGED_BOUNDING_BOX);
             completeBannerViewFutureCallback(
-                callbackDataPtr,
-                ConstantsHelper.CALLBACK_ERROR_NONE,
-                ConstantsHelper.CALLBACK_ERROR_MESSAGE_NONE);
+               callbackDataPtr,
```

Comment:
Can we run these through https://github.com/google/google-java-format ?

For bonus points let's file a ticket to add google-java-format to our code formatting script. :)

Reply:
Done.